var/www/so/capme/.inc/callback-elastic.php

```diff
@@ -224,15 +224,15 @@ function cliscript($cmd, $pwd) {
 		}
 		$timestamp = $elastic_response_object["hits"]["hits"][0]["_source"]["@timestamp"];
 		$timestamp_epoch = strtotime($timestamp);
+		$orig_ts = $timestamp_epoch;
```

Comment:
Any particular reason to create a new variable called `$orig_ts` rather than simply continuing to use the existing `$timestamp_epoch`?

Reply:
I created this when I was manipulating the epoch timestamp and left it there to make referring to the original timestamp (later on in the script) more readable, but I can remove it and just use $timestamp_epoch when referring to the original timestamp.